integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/describeTable/TestDescribeTable.scala

```diff
@@ -65,10 +68,21 @@ class TestDescribeTable extends QueryTest with BeforeAndAfterAll {
     assert(descPar.exists(_.toString().contains("Partition Parameters:")))
   }
 
+  test(testName = "Compressor Type update from carbon properties") {
+    sql("drop table if exists b")
+    sql(sqlText = "create table b(a int,b string) stored by 'carbondata'")
+    CarbonProperties.getInstance().addProperty(CarbonCommonConstants.COMPRESSOR, "gzip")
+    val result = sql(sqlText = "desc formatted b").collect()
+    assert(result.filter(row => row.getString(0).contains("Data File Compressor")).head.getString
+    (1).equalsIgnoreCase("gzip"))
+  }
+
   override def afterAll: Unit = {
     sql("DROP TABLE Desc1")
     sql("DROP TABLE Desc2")
     sql("drop table if exists a")
+    sql("drop table if exists b")
+    CarbonProperties.getInstance().addProperty(CarbonCommonConstants.COMPRESSOR, "snappy")
```

Comment:
better to use CarbonCommonConstants.DEFAULT_COMPRESSOR

Reply:
> better to use CarbonCommonConstants.DEFAULT_COMPRESSOR

+1

Reply:
done

Reply:
Please take care of this case:
` test("test compaction with different compressor for each load")` in `TestLoadDataWithCompression.scala`

create table does not specify compressor, but the global setting is changed before different loads, so one table may use multiply different compressor.

Is it leaving it blank is better? At lease filling it with any value is not correct for above case

Reply:
desc formatted just shows the compressor type that will be used for the next load and not the compressor's which have been used previously in the table

Reply:
but user may misunderstanding this. so we better distinguish it from what it was and what it will be. 

For table which has not this properties, if you want to show what compressor will be uses, need to tell user both that this table is not set compress and what the default setting is

Reply:
@qiuchenjian @zzcclp What's your opinion?